upcloud_api/api.py

```diff
@@ -86,7 +86,13 @@ def __error_middleware(self, res, res_json):
         """
         Middleware that raises an exception when HTTP statuscode is an error code.
         """
-        if res.status_code in [400, 401, 402, 403, 404, 405, 406, 409]:
+        if res.status_code >= 400:
```

Comment:
If res is `request.Response` instance `not res.ok` would also work here. Does the exact same check internally, though.